ratbagd/ratbagd-resolution.c

```diff
@@ -268,13 +378,17 @@ const sd_bus_vtable ratbagd_resolution_vtable[] = {
 	SD_BUS_VTABLE_START(0),
 	SD_BUS_PROPERTY("Index", "u", NULL, offsetof(struct ratbagd_resolution, index), SD_BUS_VTABLE_PROPERTY_CONST),
 	SD_BUS_PROPERTY("IsActive", "b", ratbagd_resolution_is_active, 0, SD_BUS_VTABLE_PROPERTY_EMITS_CHANGE),
+	SD_BUS_PROPERTY("IsDisabled", "b", ratbagd_resolution_is_disabled, 0, SD_BUS_VTABLE_PROPERTY_EMITS_CHANGE),
 	SD_BUS_PROPERTY("IsDefault", "b", ratbagd_resolution_is_default, 0, SD_BUS_VTABLE_PROPERTY_EMITS_CHANGE),
 	SD_BUS_WRITABLE_PROPERTY("Resolution", "v",
 				 ratbagd_resolution_get_resolution,
 				 ratbagd_resolution_set_resolution, 0,
 				 SD_BUS_VTABLE_UNPRIVILEGED|SD_BUS_VTABLE_PROPERTY_EMITS_CHANGE),
 	SD_BUS_PROPERTY("Resolutions", "au", ratbagd_resolution_get_resolutions, 0, SD_BUS_VTABLE_PROPERTY_CONST),
+	SD_BUS_PROPERTY("Capabilities", "au", ratbagd_resolution_get_capabilities, 0, SD_BUS_VTABLE_PROPERTY_CONST),
 	SD_BUS_METHOD("SetActive", "", "u", ratbagd_resolution_set_active, SD_BUS_VTABLE_UNPRIVILEGED),
+	SD_BUS_METHOD("SetDisabled", "", "u", ratbagd_resolution_set_disabled, SD_BUS_VTABLE_UNPRIVILEGED),
+	SD_BUS_METHOD("SetEnabled", "", "u", ratbagd_resolution_set_enabled, SD_BUS_VTABLE_UNPRIVILEGED),
```

Comment:
for things that can be enabled/disabled we should just have a `IsEnabled` read/write property. The reason we have `SetActive` is simply because only one resolution can be active so there is no `SetInactive` equivalent.

Reply:
Makes sense. Will work on it.

Reply:
Replaced it now. Can you check the get/set functions? I'm especially unsure about the message read from the set function.